File: UNIT 2/C/diskon.c
#include <stdio.h>

int main() {
double beli, diskon, bayar;
printf("jumlah pembelian = Rp. ");
scanf("%lf", &beli);
if (beli >= 100000)
diskon = 10000;
else
diskon = 0;
bayar = beli - diskon;
printf("jumlah pembelian = Rp. %.21f\n", beli);
printf("besar diskon = Rp. %.21f\n", diskon);
printf("total pembayaran - Rp. %.21f\n", bayar);
return 0;
}

File: UNIT 2/py/diskon.py
beli= int(input("Jumlah pembelian: Rp"))
if beli >= 100000:
    diskon = 10000
else:
    diskon = 0
bayar = beli - diskon

print("Total pembelian: Rp", beli)
print("Jumlah diskon: Rp", diskon)
print("Total pembayaran: Rp", bayar)